docs/example.md

```diff
@@ -128,12 +148,12 @@ You may set any other connection string options that you would like to pass to t
 
 ## ![6]({{ site.baseurl }}/assets/pce-6.png) Connection Matcher
 
-(Optional) This script defines how connections are matched.
-In most cases, the default behavior works, so you don't have to include the <span style= "font-family: courier new">connection-matcher</span> section in your \*.tdr file.
+This script is optional, and defines how connections are matched.
```

Comment:
A custom matcher should be discouraged, and in my opinion support removed entirely because of performance implications.

Reply:
I revised the text to say this: This component defines how connections are matched, and the default settings are already performance optimized. It is rare that you will need to include a custom <span style= "font-family: courier new">connection-matcher</span> section in your TDR file.